requirements/default.txt

```diff
@@ -117,9 +119,10 @@ django-jinja==2.4.1 \
 humanfriendly==4.18 \
     --hash=sha256:23057b10ad6f782e7bc3a20e3cb6768ab919f619bbdc0dd75691121bbde5591d \
     --hash=sha256:33ee8ceb63f1db61cce8b5c800c531e1a61023ac5488ccde2ba574a85be00a85
+# cfn-lint and aws-sam-translator require ~=2.6
```

Comment:
I did comments like these a while back for something--I forget what. I found they don't age well since they don't have a date or versions. So the comments tend to stay in the file long after they're obsolete.

I don't know how to make it better. Maybe adding version numbers makes it better? Maybe add a "Check this again on date YYYY-MM-DD" comment makes it better?

Reply:
Upstream knows about the issue, and it is working its way through the system. It may be ready for July. I added the issue URLs, rebased.

Reply:
:+1: